project1.html

```diff
@@ -2,40 +2,46 @@
 <head>
 	<link rel="stylesheet" type="text/css" href="project1.css">
 	<link rel="stylesheet" type="text/css" href="normalize.css">
+	
 </head>
 	<header>
 		<nav class="navbar">
 			<input class="nav-button" type="button" value="About">
 			<input class="nav-button" type="button" value="Portfolio">
+			<img src="liam.png" alt="myname">
 			<input class="nav-button" type="button" value="Contact">
+			<input class="nav-button" type="button" value="Download CV">
 		
 		</nav>
 	</header>
 <body>
 	<section class="intro-page">
-		<h1>liamlwrnc.</h1>
-		<h2>Hey I'm Liam. Welcome to my portfolio.</h2>
-	
-	
-	
-	
-	
-	
-	
-	
-	
-	
-	
-	
-	
+		<h2>Hey I'm Liam and that's Ronnie. Welcome to my portfolio.</h2>
+	</section>
 	
+	<section class="about-page">
+		<h1>About</h1>
```

Comment:
this should probably be an h2 (as well as the Contact h1) because it is not recommended to have multiple h1s on one page